sdk/src/test/java/io/dapr/it/services/HelloWorldGrpcStateService.java

```diff
@@ -0,0 +1,44 @@
+package io.dapr.it.services;
+
+import com.google.protobuf.Any;
+import com.google.protobuf.ByteString;
+import io.dapr.DaprGrpc;
+import io.dapr.DaprGrpc.DaprBlockingStub;
+import io.dapr.DaprProtos.SaveStateEnvelope;
+import io.dapr.DaprProtos.StateRequest;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+
+/**
+ * Simple example, to run:
+ * mvn clean install
+ * dapr run --grpc-port 50001 -- mvn exec:java -pl=examples -Dexec.mainClass=io.dapr.examples.Example
+ */
+public class HelloWorldGrpcStateService {
+
+  public static void main(String[] args) {
+    ManagedChannel channel =
+      ManagedChannelBuilder.forAddress("localhost", 50001).usePlaintext().build();
+    DaprBlockingStub client = DaprGrpc.newBlockingStub(channel);
+
+    String key = "mykey";
+    // First, write key-value pair.
+
+      String value = "Hello World";
+      StateRequest req = StateRequest
+        .newBuilder()
+        .setKey(key)
+        .setValue(Any.newBuilder().setValue(ByteString.copyFromUtf8(value)).build())
+        .build();
+      SaveStateEnvelope state = SaveStateEnvelope.newBuilder()
+        .addRequests(req)
+        .build();
+      client.saveState(state);
```

Comment:
For this particular example, you did not need a separate service class because it is not listening on any port, just invoking dapr. See my comment above: there is a way to start Dapr faking an app just to have it running locally.

Reply:
I can not resolve the problem using a command like CAT or something similar, so I create an empty class, that allows running DAPR without any service running. I use this approach to test HTTP state functionality.

Reply:
OK.